{{ cookiecutter.package_name }}/.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,84 @@
+# GitHub Actions workflow for testing and continuous integration.
+#
+# This file performs testing using tox and tox.ini to define and configure the test environments.
+
+name: CI Tests
+
+on:
+  push:
+    branches:
+    - master
+    tags:
+  pull_request:
+
+jobs:
+  # Github Actions supports ubuntu, windows, and macos virtual environments:
+  # https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+  ci_tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
```

Comment:
Someone kindly gave me this piece of code to allow for skipping CI

``if: "!(contains(github.event.head_commit.message, '[skip ci]') || contains(github.event.head_commit.message, '[ci skip]'))"``

Reply:
Does that still work? It didn't work for me when I tried at #11044 . In fact, I dumped the whole `github.event` context out and couldn't find any `head_commit` in it.

Reply:
I haven't tested it recently. It worked when I first added it and doesn't NOT work. But I see 11044 is more recent than my last test, so I guess it doesn't work anymore 👎 .

Reply:
i had this in the template originally, but took it out because it's no longer a reliable method of doing this. i would prefer to leave this alone until either it's supported natively (https://github.com/actions/runner/issues/774) or there's an official action published that will do this properly. a 3rd party action exists, https://github.com/marketplace/actions/ci-skip-action, and seems to work well. astropy may wish to make their own version of this if native support is not forthcoming.

Reply:
I'm pretty sure this worked for me in `photutils` when I tested it:
https://github.com/astropy/photutils/blob/master/.github/workflows/ci_tests.yml#L18

Reply:
Nope, no work astropy/photutils#1121